bot/typos.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+import random
+from rand_select import random_select_weighted_list
```

Comment:
Same issue here with the import statements
```suggestion
from .rand_select import random_select_weighted_list
```